blockFromPersonalCalendar.gs

```diff
@@ -10,33 +15,41 @@
  * - Set a trigger for an hourly run of `blockFromPersonalCalendar` 
  */
 const CONFIG = {
-  calendarId: "marianosimone@gmail.com", // (personal) calendar from which to block time
+  calendarIds: ["mattkharris@gmail.com", "aqv06aedps4ek5pggj5ug1gots@group.calendar.google.com"], // (personal) calendars from which to block time
   daysToBlockInAdvance: 30, // how many days to look ahead for
   blockedEventTitle: '❌ Busy', // the title to use in the created events in the (work) calendar
   skipWeekends: true, // if weekend events should be skipped or not
   skipAllDayEvents: true, // don't block all-day events from the personal calendar
-  workingHoursStartAt: 900, // any events ending before this time will be skipped. Use 0 if you don't care about working hours
-  workingHoursEndAt: 1800, // any events starting after this time will be skipped. Use 2300
+  workingHoursStartAt: 800, // any events ending before this time will be skipped. Use 0 if you don't care about working hours
+  workingHoursEndAt: 1900, // any events starting after this time will be skipped. Use 2300
   assumeAllDayEventsInWorkCalendarIsOOO: true, // if the work calendar has an all-day event, assume it's an OOO day, and don't block times
+  skipFreeAvailabilityEvents: true, // Ignore events that are marked as "Free" in the secondary calendar
+  color: CalendarApp.EventColor.YELLOW // set the color of any newly created events (see https://developers.google.com/apps-script/reference/calendar/event-color)
+}
+
+function blockFromPersonalCalendars() {
+  CONFIG.calendarIds.map((calendarId) => blockFromPersonalCalendar(calendarId));
```

Comment:
could you change this for a `forEach`? Given that we don't care about the result, this is more appropriate than `map`